services/server/getEvents.ts

```diff
@@ -0,0 +1,116 @@
+import client from "@/tina/client";
+
+export const getPastEvents = async (top, presenterName) => {
+  console.log("presenterName", presenterName);
+  const eventClient = await client.queries.getPastEventsQuery(
+    formatEventParams(top, presenterName)
+  );
+  return await fetchEventsWithClient(eventClient, presenterName, top);
+};
+
+export const getUpcomingEvents = async (top, presenterName) => {
+  const eventClient = await client.queries.getFutureEventsQuery(
+    formatEventParams(top, presenterName)
+  );
+  return await fetchEventsWithClient(eventClient, presenterName, top);
+};
+
+const formatEvent = (event) => {
+  const url = event.url && fixRelativeUrl(event.url);
+  return {
+    Presenter: event.presenterName,
+    Url: {
+      Description: event.title, // this is alt text for the link, ask matt what he wants to do for it
+      Url: url,
+    },
+    title: event.title,
```

Comment:
This should be an upper case "Title" to be consistent with the rest of the props

Reply:
Done - I updated this prop to "Title"